Codeherder19/Lets-Taco-Bout-It
# Feature/final component structure

### What does this PR do?
- This PR finishes the component structure and functionality and adds CSS modifications to get 100 accessibility score. Two components, TinyTaco and SavedRecipeView were created and then destroyed because I did not plan my architecture accordingly. The saved tacos can be viewed and are rendered as reused Taco components with a button to delete recipes from this screen.

### What does it fix?
- No bug to fix, this is a feature.

### Is this a feature or a fix?
- This is a feature, allowing a user to view their saved taco recipes and delete them on a button click.

### Where should the reviewer start?
- Clone down this repo, cd into the directory and run npm start.

### How should this be tested?
- Navigate through the site, cycle through random curated or wacky tacos by clicking those respective buttons. Click the save taco button and try saving a few tacos then click the view saved button to see saved recipes.
